openfisca_aotearoa/entities.py

```diff
@@ -46,4 +46,45 @@
     is_person = True,
     )
 
-entities = [Titled_Property, Person]
+Family = build_entity(
+    key = "family",
+    plural = "families",
+    label = u'Family',
+    doc = '''
+    A Family represents a collection of related persons.
+
+    Family entities are required for calculations across a number of entitlements including for example "Working for families" and "Paid Parental Leave"
+
+    A family can contain a number of roles, such as 'principal_caregiver', 'partner', 'dependant_child' & 'independant_child'.
+
+    For more information on entities, see: http://openfisca.org/doc/coding-the-legislation/50_entities.html
+    ''',
+    roles = [
+        {
+            'key': 'principal_caregiver',
+            'plural': 'principal_caregivers',
+            'label': u'Principal caregivers',
+            'doc': u'The one person who is the principal caregiver of a family.'
```

Comment:
this might be more than one person as a principal carer. In a share care arrangement, all carers with 1/3 or more of the care duties is considered a principal carer.

Section 2 here http://www.ird.govt.nz/wff-tax-credits/understanding/eligible/who-is-eligible.html